.env

```
@@ -0,0 +1,3 @@
+TODO_PORT=7540
+TODO_DBFILE=../scheduler.db
```

Файл с БД должен лежать в проекте
```suggestion
TODO_DBFILE=scheduler.db
```